site/pages/experimental-en.hbs

```diff
@@ -38,94 +39,107 @@
-		<td>v5.1</td>
-		<td>Supported via wet-boew since v6.0</td>
-	</tr>
-</tbody>
+	<thead>
+		<tr>
+			<th>Name</th>
+			<th>Description</th>
+			<th class="nowrap">Available in</th>
+			<th>Status</th>
+		</tr>
+	</thead>
+	<tbody>
+		<tr>
+			<th>CSS <code>.p-0</code></th>
+			<td>Set a padding of 0px</td>
+			<td>v5.1</td>
+			<td>Supported via wet-boew since v6.0</td>
+		</tr>
+		<tr>
+			<th>CSS <code>.mb-sm-5</code></th>
+			<td>Set a margin bottom of 50px for view port from small screen and up</td>
+			<td>v5.1</td>
+			<td>Supported via wet-boew since v6.0</td>
+		</tr>
+		<tr>
+			<th>CSS <code>.p-sm-3</code></th>
+			<td>Set a padding of 15px for view port from small screen and up</td>
+			<td>v5.1</td>
+			<td>Supported via wet-boew since v6.0</td>
+		</tr>
+		<tr>
+			<th>CSS <code>.px-sm-3</code></th>
+			<td>Set a left and right padding of 15px for view port from small screen and up</td>
+			<td>v5.1</td>
+			<td>Supported via wet-boew since v6.0</td>
+		</tr>
+		<tr>
+			<th>CSS <code>.bg-darker</code></th>
+			<td>Full black background color</td>
+			<td>v5.1</td>
+			<td></td>
+		</tr>
+		<tr>
+			<th>CSS <code>.bg-gctheme</code> and <code>.bg-gctheme.well.header-rwd</code></th>
+			<td>GCWeb theme default background color</td>
+			<td>v5.1</td>
+			<td></td>
+		</tr>
+		<tr>
+			<th>CSS <code>.bg-pnkDy</code> and <code>.bg-pnkDy.well.header-rwd</code></th>
+			<td>Pink day background color</td>
+			<td>v5.1</td>
+			<td></td>
+		</tr>
+		<tr>
+			<th>CSS <code>.pnkDy-theme</code> set global</th>
+			<td>Apply global template color change for the pink day color, as that property is defined on the <code>&lt;html&gt;</code> or <code>&lt;body&gt;</code> element</td>
+			<td>v5.1</td>
+			<td></td>
+		</tr>
+		<tr>
+			<th>CSS <code>.dark-theme</code> set global</th>
+			<td>Apply global template color change for full black, as that property is defined on the <code>&lt;html&gt;</code> or  <code>&lt;body&gt;</code> element</td>
+			<td>v5.1</td>
+			<td></td>
+		</tr>
+		<tr>
+			<th>CSS <code>.bg-cover</code></th>
+			<td>Background size cover</td>
+			<td>v5.1</td>
+			<td></td>
+		</tr>
+		<tr>
+			<th>Plugin <code>[data-bgimg]</code></th>
+			<td>Take the URL value and set it as the background image. To be replaced by CSS4 selector.</td>
+			<td>v5.1</td>
+			<td></td>
+		</tr>
+		<tr>
+			<th>CSS <code>.bg-img-hdng</code></th>
+			<td>Add a background image to a heading.</td>
+			<td>v5.1</td>
+			<td></td>
+		</tr>
+		<tr>
+			<th>CSS <code>.stretched-link</code></th>
+			<td>Propagates the clickable area around an hyperlink tag up to a relatively positioned parent.</td>
+			<td>v5.1</td>
+			<td>Supported via wet-boew since v6.0</td>
+		</tr>
+		<tr>
+			<th>CSS:
+				<ul>
+					<li>Danger alert: <code>.alert-danger</code></li>
+					<li>Warning alert: <code>.alert-warning</code></li>
+					<li>Success alert: <code>.alert-success</code></li>
+					<li>Info alert: <code>.alert-info</code></li>
```

Comment:
@duboisp Should these mention the ``.provisional`` class anywhere? Or is that already implied for all classes this page covers?

Reply:
It is implied.
Can you do 4 row instead of one?

Reply:
Updated in latest commit.